dev-libs/expat/patches/expat-2.2.2.patchset

```diff
@@ -0,0 +1,25 @@
+From 285c3be03c49359ee2d67c3f5fd2a7a0326acafa Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Fri, 21 Jul 2017 12:17:21 +0200
+Subject: fix missing arc4random for Haiku
+
+
+diff --git a/lib/xmlparse.c b/lib/xmlparse.c
+index c430541..69fe822 100644
+--- a/lib/xmlparse.c
++++ b/lib/xmlparse.c
+@@ -49,6 +49,11 @@
+ # include <bsd/stdlib.h>
+ #endif
+ 
++#ifdef __HAIKU__
++#if !defined(XML_POOR_ENTROPY)
++#define XML_POOR_ENTROPY 1
```

Comment:
So until now, we had already a poor entropy, but weren't aware of it. Thus this PR doesn't change the situation (no regression). Is it right?